#170947430 Setup docker

RusimbiPatrick opened the pull request:
#### What does this PR do?
Have Docker configured
#### Description of Task to be completed?

- Setting up Dockerfile
- Setting up docker-compose
- Creating a script for running docker

#### How should this be manually tested?

- Install docker on a local machine
- run `npm run docker`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[Pivotal tracker](https://www.pivotaltracker.com/story/show/170947430)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

AbdoulNuru commented on Dockerfile:
```diff
@@ -0,0 +1,25 @@
+EXPOSE 3000
+
+# Define commands that will run the app
+CMD ["npm", "start"]
```
@RusimbiP Add a new Line at the end of the file

RusimbiP replied:
> @RusimbiP Add a new Line at the end of the file

Thanks

RusimbiPatrick replied:
Thanks

izzett222 committed:
chore(heroku): deploy on heroku (#8)

- add the procfile
- create heroku staging and production app
- add code maintainability badge and the build badge on readme

[Finishes #170947431]

kcharles52 commented on .dockerignore:
```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```
```suggestion
npm-debug.log

```

RusimbiPatrick committed:
chore(docker):setup docker
-remove unnecesary variable
[Maintains #170947430]
